slides/advriskmin/slides-advriskmin-regression-l2.tex

```diff
@@ -112,15 +112,15 @@
 
 For the optimal constant model $\fx = \theta$ for the L2-loss $\Lxy = \left(y - \fx\right)^2$ we solve the optimization problem 
 $$
-\argmin_{f \in \Hspace} \riskef = \argmin_{c \in \R} \sumin (\yi - \theta)^2. 
```

Comment:
It should be Theta

![image](https://user-images.githubusercontent.com/35375828/200591833-a37c07fd-d807-4720-b87d-b1a974640889.png)

Reply:
:+1: